src/Network/Network.Test/ScenarioTests/ExpressRoutePortTests.cs

```diff
@@ -25,7 +25,7 @@ public ExpressRoutePortTests(Xunit.Abstractions.ITestOutputHelper output)
         {
         }
 
-        [Fact]
+        [Fact(Skip="No bandwidth available yet")]
```

Comment:
@cormacpayne Tests for `ExpressRoutePorts` are failing because no bandwidth is available from provider in tests. I'm trying to reach out to the person who worked on them so they could take a look.

Is it possible to merge this PR with two skipped tests to unblock other PRs? We'll make sure they are re-enabled before merging into `master`